cdk/stateful-api-stack.ts

```diff
@@ -0,0 +1,58 @@
+import { Construct, Stack, StackProps } from "@aws-cdk/core";
+import { Domain, ElasticsearchVersion } from '@aws-cdk/aws-elasticsearch';
+import { AnyPrincipal, Effect, PolicyStatement } from "@aws-cdk/aws-iam";
+import { BastionHostLinux, Vpc } from '@aws-cdk/aws-ec2';
+
+interface StatefulApiStackProps extends StackProps {
+  stage: string
+  esNodeType?: string
+}
+
+export default class StatefulApiStack extends Stack {
+  public readonly elasticsearchDomain: Domain;
+
+  constructor(scope: Construct, id: string, props: StatefulApiStackProps) {
+    const {
+      stage,
+      esNodeType = 't2.small.elasticsearch',
+      ...superProps
+    } = props;
+
+    super(scope, id, {
+      stackName: `${stage}-obp-cdk-stateful-api`,
+      description: `Stateful API stack for the "${stage}" stage`,
+      terminationProtection: true,
+      ...superProps
+    });
+
+    const vpc = Vpc.fromLookup(this, 'Vpc', { isDefault: true });
```

Comment:
If I understand this correctly we're still putting everything in the default VPC?